src/TokenLogicFunding_20230919/TokenLogicFunding_20230919.t.sol

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {GovHelpers} from 'aave-helpers/GovHelpers.sol';
+import {AaveGovernanceV2} from 'aave-address-book/AaveGovernanceV2.sol';
+import {ProtocolV2TestBase} from 'aave-helpers/ProtocolV2TestBase.sol';
+import {AaveV3Ethereum, AaveV3EthereumAssets} from 'aave-address-book/AaveV3Ethereum.sol';
+import {IERC20} from 'solidity-utils/contracts/oz-common/interfaces/IERC20.sol';
+import {TokenLogicFunding_20230919} from './TokenLogicFunding_20230919.sol';
+
+contract TokenLogicFunding_20230919_Test is ProtocolV2TestBase {
+  function setUp() public {
+    vm.createSelectFork(vm.rpcUrl('mainnet'), 18166945);
+  }
+
+  function testpayload() public {
+
+    address TOKENLOGIC = 0x3e4A9f478C0c13A15137Fc81e9d8269F127b4B40;
+    address GHO_WHALE = 0xBA12222222228d8Ba445958a75a0704d566BF2C8;
+    uint256 STREAM_ID = 100012;
```

Comment:
better to not guess the stream_id, and get it from contract

Reply:
streamID is not available anywhere that's easy to get. Emitting an event on the payload and then reading that value is not really easy without adding an extra dependency to the repo (stdutils from forge)

I could make the execute function return a value but that would break the inheritance.

I found no good clean solution here so I simulated the payload execution and copy pasted the value.

Please let me know if you find any of my solutions suitable, or if you have better ones that I missed